src/selectedtests/task_mappings/create_task_mappings.py

```diff
@@ -419,7 +420,14 @@ def _create_task_map(tasks: List[Task]) -> Dict:
     :param tasks: List of tasks to map.
     :return: Dictionary of tasks by display_name.
     """
-    return {task.display_name: task for task in tasks}
+    execution_tasks_map = {
+        task: task
+        for task in itertools.chain.from_iterable(
+            task.json["execution_tasks"] for task in tasks if "execution_tasks" in task.json
```

Comment:
[nit] There might be some logic here we would want to pull into evergreen.py at some point. I can imagine other places where this might be useful.

Reply:
I'm going to leave that to another PR. evergreen.py doesn't expose execution_tasks as a top level attribute at the moment.